test/manifolds/sphere.jl

```diff
@@ -145,10 +145,10 @@ using ManifoldsBase: TFVector
         @test repr(M) == "Sphere(2, ℍ)"
         @test typeof(get_embedding(M)) === Euclidean{Tuple{3},ℍ}
         @test representation_size(M) == (3,)
-        p = [Quaternion(1.0), Quaternion(1.0im), Quaternion(0.0, 0.0, -1.0, 0.0)]
+        p = [Quaternion(1.0), Quaternion(1.0), Quaternion(0.0, 0.0, -1.0, 0.0)]
```

Comment:
Restores original point:
```suggestion
        p = [Quaternion(1.0), Quaternion(0, 1.0, 0, 0), Quaternion(0.0, 0.0, -1.0, 0.0)]
```